Networking/Networking/Remote/SitePluginsRemote.swift

```diff
@@ -13,7 +13,12 @@ public class SitePluginsRemote: Remote {
     public func loadPlugins(for siteID: Int64,
                             completion: @escaping (Result<[SitePlugin], Error>) -> Void) {
         let path = Constants.sitePluginsPath
-        let request = JetpackRequest(wooApiVersion: .none, method: .get, siteID: siteID, path: path, parameters: nil)
+        let request = JetpackRequest(wooApiVersion: .none,
+                                     method: .get,
+                                     siteID: siteID,
+                                     path: path,
+                                     parameters: nil,
+                                     availableAsRESTRequest: true)
```

Comment:
The PR description mentioned that other endpoints aren't converted because we're not supporting Jetpack installation for REST login. I remember we implemented WooCommerce plugin installation, does that use a different endpoint? Do we plan to support Jetpack installation in the future, with a similar upsell to JCP sites?

Reply:
We implemented Woo installation in a web view only - we want to do that natively but haven't gotten around to doing that yet. Similarly, we want to support Jetpack installation after the first iteration of the REST API project is shipped, so we'll migrate the rest of the endpoints then.